CMakeLists.txt

```diff
@@ -187,14 +185,21 @@ endif()
 set(DLL_PATH_LIST
     "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_LIBRARY_DIRS}/../bin")
 
-if(UMF_BUILD_LIBUMF_POOL_SCALABLE)
-    pkg_check_modules(TBB tbb)
-    if(NOT TBB_FOUND)
-        find_package(TBB REQUIRED tbb)
-    endif()
+pkg_check_modules(TBB tbb)
+if(NOT TBB_FOUND)
+    find_package(TBB OPTIONAL_COMPONENTS tbb)
+endif()
+if(TBB_FOUND OR TBB_LIBRARY_DIRS)
     # add PATH to DLL on Windows
     set(DLL_PATH_LIST
         "${DLL_PATH_LIST};PATH=path_list_append:${TBB_LIBRARY_DIRS}/../bin")
+    set(UMF_ENABLE_POOL_SCALABLE TRUE)
```

Comment:
Maybe `UMF_ENABLED_POOL_SCALABLE` or `UMF_POOL_SCALABLE_ENABLED` ?

Reply:
done, changed to UMF_POOL_SCALABLE_ENABLED